### ISSUE TITLE
Public Enum Obfuscation

### ISSUE THREAD

**Issue by joelpjy** [rockbruno/swiftshield#92]
Before Obfuscation:
```
public enum Car {
    case TOYOTA

    public var value: String {
        switch self {
        case .TOYOTA: return "toyota"
        }
    }
}
```

After obfuscation:
```
public enum Car {
    case AUciwCDDxRhUHreXgAmFMPzeSlJFHrVn

    public var value: String {
        switch self {
        case .AUciwCDDxRhUHreXgAmFMPzeSlJFHrVn: return "toyota"
        }
    }
}
```
Is it possible for public enum to stay un-obfuscated? (Im using building an sdk project, using the "-sdk" parameter)

**Comment by awaisiosdev** [comment]
I have the same problem. if there is something we can do to fix it that would be great.

**Comment by CanfieldMobileDev** [comment]
Same problem here

**Comment by rockbruno** [comment]
Unfortunately that's a problem with SourceKit, so there's nothing we can do besides jump inside the Swift Compiler or report it to Apple. The same happens with public extensions.

**Comment by rockbruno** [comment]
Fixed in 4.0.2
